core/src/main/scala/kafka/network/RequestConvertToJson.scala

```diff
@@ -101,6 +101,7 @@ object RequestConvertToJson {
       case req: ConsumerGroupDescribeRequest => ConsumerGroupDescribeRequestDataJsonConverter.write(req.data, request.version)
       case req: ControllerRegistrationRequest => ControllerRegistrationRequestDataJsonConverter.write(req.data, request.version)
       case req: AssignReplicasToDirsRequest => AssignReplicasToDirsRequestDataJsonConverter.write(req.data, request.version)
+      case req: ListClientMetricsResourcesRequest => ListClientMetricsResourcesRequestDataJsonConverter.write(req.data, request.version)
```

Comment:
Seems the sequence of definitions earlier was alphabetical and got messed recently. Not sure if we should place this in the end or try to maintain alphabetical order, though good to have some order.

Reply:
Yes, it's mostly ordered with the latest added to the end. I've ordered the lists. Thanks.